packages/p2p/src/stores/general-store.js

```diff
@@ -618,6 +624,7 @@ export default class GeneralStore extends BaseStore {
             this.setIsListed(!!p2p_advertiser_info.is_listed);
             this.setNickname(p2p_advertiser_info.name);
             this.setUserBlockedUntil(p2p_advertiser_info.blocked_until);
+            this.setUserBlockedCount(p2p_advertiser_info.blocked_by_count);
```

Comment:
can we destructure `p2p_advertiser_info` so we don't have to call it multiple times?

`const { blocked_until, blocked_by_count, id, is_approved, is_blocked, is_listed, name} = p2p_advertiser_info;`